extensions/amp-base-carousel/validator-amp-base-carousel.protoascii

```diff
@@ -56,6 +56,12 @@ attr_lists: {
     # Media query / positive integer pairs
     value_regex: "([^,]+\\s+(\\d+),\\s*)*(\\d+)"
   }
+  attrs: {
+    name: "controls"
+    value: "always"
+    value: "auto"
+    value: "never"
```

Comment:
```suggestion
    # Media query / (always|auto|never)
    value_regex: "([^,]+\\s+(always|auto|never),\\s*)*(always|auto|never)"
```

Reply:
This accounts for media query?

Reply:
Yes, it's the same syntax as all other responsive attributes in this file.